client/libs/theme/_fonts.scss

```diff
@@ -17,8 +17,11 @@
 
 // Text Color
 .primary-text { color: $primary !important; }
+.bg-primary { background-color: $primary !important; }
```

Comment:
Create a new section called `Background Color` and put this class under it. Create other ones with our variables.